lib/templates/android_screen_base.tt

```diff
@@ -183,4 +183,41 @@ class AndroidScreenBase < Calabash::ABase
     # Clicking in the Done button
     touch "* id:'button1'"
   end
+  
+  # @param query [String] query that will be searched
+  # @param scroll_location_id [String]
+  # @param scroll_location_query [String, nil]
+  # @param direction [Symbol]
+  # @param options [Hash]
+  # @option options [String] :timeout
+  # @option options [String] :flick in case we need flick
+  # @todo #TODO Add reset option to allow going to top position
+  # before start search
+  # Exemple -> scroll_looking_for_query(query_text_variable, nil, scroll_location_query_variable, :down, :timeout=>7)
+
+  def scroll_looking_for_query query, scroll_location_id, scroll_location_query = nil , direction= :down, options = {}
```

Comment:
Additionally, here is the `flick` definition by Xamarin:
https://developer.xamarin.com/guides/testcloud/calabash/working-with/gestures/